democracy/views/section_comment.py

```diff
@@ -231,6 +236,16 @@ def get_deleted_by_type(self, obj):
         # No information about who deleted the comment
         return "unknown"
 
+    @cached_property
+    def request(self):
+        return self.context.get('request', None)
```

Comment:
Unnecessary default value.
```suggestion
        return self.context.get('request')
```